src/components/icons/GreenCircle.tsx

```diff
@@ -0,0 +1,10 @@
+import { SVG, type SVGWrapperProps } from "../SVG"
+
+export default function GreenCircle(props: SVGWrapperProps) {
+    return (
+        <SVG viewBox="0 0 16 16" {...props}>
+            <rect width="16" height="16" rx="8" fill="#61DC75" fill-opacity="0.3" />
```

Comment:
Make sure to replace svg props like "fill-opacity" with their react counterpart in camel case, "fillOpacity"